packages/react-widgets/src/Calendar.tsx

```diff
@@ -430,8 +427,8 @@ function Calendar({
 
   const handleKeyDown = (e: React.KeyboardEvent<HTMLDivElement>) => {
     let ctrl = e.ctrlKey || e.metaKey
-    let key = e.key
-    let direction = ARROWS_TO_DIRECTION[key]
+    let key = e.key;
+    let direction : Direction = (ARROWS_TO_DIRECTION as any)[key]
```

Comment:
does `key as Direction` not work in the index signature?

Reply:
Fixed